smartpy/losses/classification_losses.py

```diff
@@ -0,0 +1,16 @@
+from smartpy.interfaces.loss import Loss
+
+import theano.tensor as T
+
+
+class NegativeLogLikelihood(Loss):
+    def _loss_function(self, model_output):
+        nll = -T.log(model_output)
+        indices = T.cast(self.target[:, 0], dtype="int32")  # Targets are floats.
+        selected_nll = nll[T.arange(self.target.shape[0]), indices]
```

Comment:
What about ``self.target.T[0]`` ? Should be faster than accessing *n* times the matrix.
Or ``self.target[:,0].reshape(self.target.shape[0])``